fiftyone/server/routes/view.py

```diff
@@ -0,0 +1,34 @@
+"""
+FiftyOne Server ``/view`` route.
+| Copyright 2017-2022, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+import glob
+import os
+import numpy as np
+
+from starlette.endpoints import HTTPEndpoint
+from starlette.requests import Request
+
+import eta.core.serial as etas
+import fiftyone.server.view as fosv
+
+import fiftyone as fo
+from fiftyone.server.decorators import route
+
+
+class View(HTTPEndpoint):
```

Comment:
curious when we would be using the http endpoint 🤔

Reply:
I was initially thinking of using endpoints to trigger the SDK functions, but there is probably a more acceptable way of triggering `fo.load_dataset` from the app...

@benjaminpkane Do you have any recommendations? Is another way of calling the SDK functions Brian already wrote using graphQL? Or is the only other option to not use 'load_view' and 'save_view' and just implement the mutations/queries for saved views directly?